### DISCUSSION TITLE
Radio Reception R link 1

### DISCUSSION THREAD

**Discussion by BGMark1** [cedricp/ddt4all#1282] [General]
Hello everyone!

I recently retrofitted an R-Link system into my Megane RS 250. Everything is working smoothly—except the radio reception for both FM1 and FM2, which is completely non-existent. Initially, it worked, but after an update, the signal disappeared. When I checked in DDT, I noticed that all the radio parameters are set to 0.

To troubleshoot, I tested with another R-Link unit intended for a different retrofit, but the same issue persisted: no radio reception, and all parameters are still 0.

If anyone has an R-Link system with functional analog radio reception, could you please share your radio parameters with me? Any other ideas or suggestions to get this fixed would also be greatly appreciated. Thanks in advance!
![IMG_1565](https://github.com/user-attachments/assets/71a19d1d-24e5-4907-b73c-6370299dfe19)

**Comment by Jeffreyvanesch** [comment]
I'll need to take a look at the params set in my R-Link1 for the analog FM. 
I've recently had my R-Link unit replaced by the dealer because it was producing an almost jackhammer like sound at maximum volume on completely random times.

**Reply by BGMark1** [reply]
That would be great. I’m suspecting the radio not to know what to do when trying to use FM. 
![IMG_1585](https://github.com/user-attachments/assets/3685da67-4814-4648-88c9-6d78688be8b8)
